packages/flutter_tools/lib/src/ios/xcode_build_settings.dart

```diff
@@ -74,7 +74,12 @@ void _updateGeneratedXcodePropertiesFile({
     : project.ios.generatedXcodePropertiesFile;
 
   generatedXcodePropertiesFile.createSync(recursive: true);
-  generatedXcodePropertiesFile.writeAsStringSync(localsBuffer.toString());
+  final String originalContents = generatedXcodePropertiesFile.readAsStringSync();
+  final String newContents = localsBuffer.toString();
+  if (originalContents != newContents) {
+    globals.logger.printStatus('Updating xccconfig from:\n$originalContents\n to \n$newContents');
```

Comment:
I'm going to remove this, only added to see why CI is failing.

Reply:
```
Fingerprint write error: FileSystemException: Cannot open file, path = 'build/ios/pod_inputs.fingerprint' (OS Error: No such file or directory, errno = 2)
```
`writeAsStringSync` isn't recursively creating the directory structure.
https://github.com/flutter/flutter/blob/ca6cecf034d9c18fcb6ea70309cc855e353aa0f8/packages/flutter_tools/lib/src/base/fingerprint.dart#L64-L65